packages/compile-solidity/run.js

```diff
@@ -331,6 +331,7 @@ function processContracts({
   originalSourcePaths,
   solcVersion
 }) {
+  if (!compilerOutput.contracts) return [];
```

Comment:
Should we play it safe and also handle `compilerOutput.contracts` being `[]`?

Reply:
Sounds reasonable.

Reply:
Actually now that I think about it this seems weird as the signature for the method clearly expects an object.

Reply:
Oh, I just figured it's the sort of situation where there's no real downside to playing it safe. Your call, my approval still stands!